Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/components/batteryInput/index.tsx

```diff
@@ -1,8 +1,9 @@
-const BatteryInput = () => {
+const BatteryInput = (props) => {
   return (
     <div className="flex w-full flex-col items-center gap-2">
       <label>Battery Percentage (%):</label>
```

Comment:
In App.tsx, you give the battery an intial value of 0, but when the app mounts for the first time, there is no initial value for the battery input. 

Why is this? Look into controlled input fields work in React